core/tests/build_sources/mock_tsc_output_log.txt

```diff
@@ -0,0 +1,4 @@
+c5:26:51 AM - Starting compilation in watch mode...
```

Comment:
@ankita240796, is this an expected change? (Do we need this file?)

Reply:
Yes, we need this for testing.

Reply:
(Ignore this comment)*